### ISSUE TITLE
Builds are failing because of not enough funds to send tx

### ISSUE THREAD

**Issue by nberger** [windingtree/LifToken#134]
I guess the creation of the Market Maker from the Crowdsale increased the funds spending so much that the builds start to fail. Example error:

```
1) Contract: LifToken "before each" hook for "should throw an error when trying to transfer more than balance":
     Error: sender doesn't have enough funds to send tx. The upfront cost is: 471238800000000000 and the senders account only has: 467012400000000000
```

Failing build example: https://travis-ci.org/windingtree/LifToken/builds/269134690

/cc @AugustoL

**Comment by AugustoL** [comment]
im on it,

**Comment by nberger** [comment]
Hopefully fixed by #135 + #142 + #143

**Comment by nberger** [comment]
This seems to be fixed now :tada:, closing!
